plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/worker/KtLintWorkAction.kt

```diff
@@ -123,6 +123,7 @@ abstract class KtLintWorkAction : WorkAction<KtLintWorkAction.KtLintWorkParamete
                     results.add(result)
                 }
             } catch (e: RuntimeException) {
+                logger.error(e.message)
```

Comment:
Doesn't the logger have a method that takes an exception? Is there a reason you aren't logging the stack trace?

Reply:
because a GradleException is thrown, which will contain the stacktrace, and print it when the build is run with `--stacktrace` and I think the expectation would be that the stacktrace only prints in that scenario